### PR TITLE
Remove Shipping Setup of Wizard

### PR DESCRIPTION
**Pull request by mikejolley**
With the introduction of shipping zones in 2.6, for the wizard to remain useful we could take 1 of 2 paths.
1. Include a zone editor in the wizard
2. Remove it and prompt the user to make a zone, like we prompt them to make a product

I've avoided 1 because it introduces code duplication, complexity, and its a non-interface that the user has to learn during setup.

This PR implements 2. 
- You can enable shipping during wizard.
- Post-wizard you create products as normal
- **When you have a product** and no shipping rates, and shipping is enabled, **then** a notice will appear stating you need to add some shipping methods.

2016-03-31_at_17 57_720

By showing the notice after product creation, it prevents information overload if they are new to the platform, without them wondering why shipping doesn't work.

@allendav @claudiosmweb @justinshreve

### CODE REVIEW

File: assets/css/activation.scss [hunk 1]
```diff
@@ -83,7 +83,7 @@ p.woocommerce-actions,
 	margin-bottom: 1em !important;
 }
 
-div.woocommerce-legacy-shipping-notice {
+div.woocommerce-legacy-shipping-notice, div.woocommerce-no-shipping-methods-notice {
```
Comment (justinshreve) [inline assets/css/activation.scss]: Each selector should be on it's own line.

File: assets/js/admin/wc-setup.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ jQuery(function( $ ) {
 
 	var locale_info = $.parseJSON( wc_setup_params.locale_info );
 
-	$('select[name="store_location"]').change(function(){
+	$( 'select[name="store_location"]' ).change(function(){
```
Comment (justinshreve) [inline assets/js/admin/wc-setup.js]: There should be a space before `function()`. Ditto for line 28.

**Comment by justinshreve** [comment]
:+1: to implementing method 2. Code LGTM.

**Comment by claudiosmweb** [comment]
LGTM.

**Pull request merged**